src/observe-tag.js

```diff
@@ -78,26 +86,21 @@ module.exports = tagFunction => {
 
 		// if oldTag was defined, then we need to replace it with the new result
 		if (oldTag) {
-			oldTag.replaceWith(tagResult)
-
 			// if an element had focus, reapply it
+			let elementToGiveFocus
 			if (removedElementWithFocusData.index >= 0) {
-				const allElements = parentAndChildrenElements(tagResult)
+				const allActiveLikeElements = parentAndChildrenElements(tagResult, removedElementWithFocusData.tagName)
```

Comment:
rather than filtering after the fact, we get only elements with the same tag name (this is why they are `active-like`